drivers/avr/spi_master.h

```diff
@@ -39,19 +39,15 @@ typedef int16_t spi_status_t;
 #ifdef __cplusplus
 extern "C" {
 #endif
-void spi_init(void);
-
-bool spi_start(pin_t slavePin, bool lsbFirst, uint8_t mode, uint16_t divisor);
 
```

Comment:
?

Reply:
honestly, not sure. Merge issue probably.   Also, definitely in progress